cosmic_ray/testing/test_runner.py

```diff
@@ -66,4 +68,4 @@ def __call__(self):
                                   test_result[1])
         except Exception as e:
```

Comment:
Side note: for some projects I'm using landscape.io as analytics tool. Here's the report for CR:
https://landscape.io/github/MrSenko/cosmic-ray/9/messages

I will go ahead and clean up some of these later.